.copr/make-srpm.sh

```diff
@@ -3,6 +3,12 @@
 set -eux
 
 outdir="$1"; shift
+mkdir -p "$outdir"
+
+# when this is specified, build a source directory and skip the rpmbuild step
+if [ "${1:-}" = "srcdir" ]; then
+    srcdir=1
+fi
```

Comment:
It would be cleaner to instead split out the "prepare sources" part into a separate script. Something like [this](https://github.com/fedora-selinux/selinux-policy/commit/d36321e611f97911c0cc1a9f7b7d5aaad002be60) (untested, may contain mistakes; feel free to squash into your commit).

Reply:
Indeed, looks good, thanks! It was almost working. I fixed it up, squashed it, and updated the commit message.

FTR, feel free to force-push over this, "Allow edits and access to secrets by maintainers" is quite deliberate. Especially as I'll be on PTO from tomorrow on.

Reply:
> FTR, feel free to force-push over this, "Allow edits and access to secrets by maintainers" is quite deliberate.

Yes, but I find it a bit inappropriate to force-push bigger changes like that. I would use it only for minor corrections/typos.